configs/real_basicvsr/realbasicvsr_wogan-c64b20-2x30x8_8xb2-lr1e-4-300k_reds.py

```diff
@@ -275,9 +275,10 @@
 
 # optimizer
 optim_wrapper = dict(
-    constructor='DefaultOptimWrapperConstructor',
-    type='OptimWrapper',
-    optimizer=dict(type='Adam', lr=1e-4, betas=(0.9, 0.99)))
+    constructor='MultiOptimWrapperConstructor',
```

Comment:
You have only one OptimWrapper. Is it reasonable to use MultiOptimWrapperConstructor?

Reply:
I will modify in later PR